documentation/examples/prometheus-agent.yml

```diff
@@ -0,0 +1,22 @@
+# my global config
+global:
+  scrape_interval: 15s # Set the scrape interval to every 15 seconds. Default is every 1 minute.
+  evaluation_interval: 15s # Evaluate rules every 15 seconds. The default is every 1 minute.
```

Comment:
Hmmm, since there is no rule evaluation, should `evaluation_interval` also be a reason for failure? 🤔

Reply:
I don't think we should go that far